Common/ArmoniK.DevelopmentKit.Common/ArmoniK.DevelopmentKit.Common.csproj

```diff
@@ -20,27 +20,65 @@
     <LangVersion>latest</LangVersion>
     <SignAssembly>True</SignAssembly>
     <AssemblyOriginatorKeyFile>../../kp.snk</AssemblyOriginatorKeyFile>
+    <Version>1.0.0-local</Version>
+    <Authors>ANEO</Authors>
+    <Company>ANEO</Company>
+    <Product>ArmoniK</Product>
+    <Description>ArmoniK Package SDK for high level API</Description>
+    <Copyright>Copyright © ANEO 2022</Copyright>
+    <Summary>ArmoniK Package SDK for high level API</Summary>
+    <NeutralLanguage>en-US</NeutralLanguage>
+    <Title>$(MSBuildProjectName)</Title>
+    <PackageId>$(MSBuildProjectName)</PackageId>
+    <PackageTags>armonik</PackageTags>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <PackageIcon>armonik-logo.png</PackageIcon>
+    <!--<PackageIconUrl></PackageIconUrl>-->
+    <!--<PackageProjectUrl></PackageProjectUrl>-->
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <RootNamespace>$(MSBuildProjectName)</RootNamespace>
+    <AssemblyName>$(MSBuildProjectName)</AssemblyName>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    
+    <IncludeSymbols>true</IncludeSymbols>
+    
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <!--Package Validation-->
+    <EnablePackageValidation>true</EnablePackageValidation>
+    <Deterministic>true</Deterministic>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="**\*.orig" />
+    <None Include="../../LICENSE" Pack="true" PackagePath="LICENSE.md" />
+    <None Include="../../armonik-logo.png" Pack="true" PackagePath="\" />
+    <None Include="../../README.md" Pack="true" PackagePath="\" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="6.0.0" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All" Version="1.1.1" />
+  </ItemGroup>
+
   <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
     <DebugType>Embedded</DebugType>
     <IncludeSymbols>true</IncludeSymbols>
     <DefineConstants>DEBUG;TRACE</DefineConstants>
   </PropertyGroup>
 
-  <Target Name="BeforeBuild">
-  <Message Importance="High" Text="This the framework sss $(TargetFramework)" />
-  </Target>
+  <PropertyGroup Condition="'$(Configuration)'=='Release'">
```

Comment:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/code-generation#optimize

> The Optimize option is enabled by default for a Release build configuration.

Reply:
Good to know. I think VS add it when I switch to release once